dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -963,9 +963,9 @@ def generate_teacher_student(name, teacher_authorized)
   email, password = generate_user(name)
 
   steps %Q{
-    Given I am on "http://studio.code.org/reset_session"
```

Comment:
how do we know that we don't need to hit "reset_session" here first?

Reply:
Turns out we do need to hit "reset_session" for tests like Gamelab Flow that alternate between signed in students and signed in teachers. I added it back in to the step.